### DISCUSSION TITLE
on "Read more"

### DISCUSSION THREAD

**Discussion by panva** [community/community#6274] [Other Feature Feedback, Questions, & Ideas] [answered]
In the current UI when I navigate to the `/releases` view I can actually page search for a thing I need knowing that if it's not found I need to look on the next page. In the new UI I am forced to open every single release when looking for something since the individual release notes are truncated.

In that regard I don't find the new UI that helpful and it's what makes me disable the preview.

**Comment by damianstasik** [comment]
I just disabled the new UI because of that, please consider adding expanding/collapsing instead of having to open a new page.

**Comment by MylesBorins** [comment, marked as answer]
Please see my update in one of the other discussions about this topic. We are working on a number of fixes

https://github.com/github/feedback/discussions/5979#discussioncomment-1449589

**Reply by MylesBorins** [reply]
Just FYI we've shipped the majority of the fixes I mentioned in this comment
